Source/Modules/csharp.cxx

```diff
@@ -1434,6 +1514,22 @@ class CSHARP:public Language {
     return SWIG_OK;
   }
 
+  /* -----------------------------------------------------------------------
+ * doxygenComment()
+ * Simply translates the doxygen comment and places it into the appropriate
```

Comment:
I'm not sure if "translates" here is really correct, as it doesn't seem to be doing any translation, does it?

Reply:
It depends on the translator, just default being verbatim copy. I mean, technically C# module knows that the translator is doing nothing only in module constructor (that may change if someone will introduce an alternative).

Reply:
It depends on the translator, just default being verbatim copy. I mean, technically C# module knows that the translator is doing nothing only in module constructor (that may change if someone will introduce an alternative).